node/windows-packaging/CalicoWindows/node/node-service.ps1

```diff
@@ -118,6 +118,10 @@ if ($env:CALICO_NETWORKING_BACKEND -EQ "windows-bgp" -OR $env:CALICO_NETWORKING_
     }
 }
 
+# Start k8s services
+Start-Service kubelet
+Start-Service kube-proxy
+
```

Comment:
I think we shouldn't use node-service to start kubelet/kube-proxy services.

Reply:
I agree that this is not a cool solution, but if we start kubelet before primary network adapter has been prepared it might use wrong adapter to start pods.
It was a reason for the PR.

When we restart a node, we have hyper-v network adapter instead of native adapter but we don't know if it is new or old one. CalicoNode service will remove old network adapters. So, if kubelet starts a pod with it the pod will be broken when CalicoNode completes their job.

Reply:
@song-jiang I change the strategy. Now `kubelet` is awaiting for the calico network initialized.
What do you think about this way?